src/style.css

```diff
@@ -0,0 +1,43 @@
+body {
+  margin: 0;
+}
+
+.stars__star {
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 16px;
+  width: 16px;
+  background-image: url(./images/star.svg);
+}
+
+.stars--5 .stars__star {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--0 .stars__star:nth-child(-n + 5) {
+  background-image: url(./images/star.svg);
+}
+
+.stars--1 .stars__star:nth-child(-n + 1) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--2 .stars__star:nth-child(-n + 2) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--3 .stars__star:nth-child(-n + 3) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--4 .stars__star:nth-child(-n + 4) {
+  background-image: url(./images/star-active.svg);
+}
```

Comment:
Need to fix it
<img width="1133" alt="image" src="https://github.com/mate-academy/layout_stars/assets/64750958/db2ad954-753c-4119-a8ea-c06f855beff2">